Musico/db/migrate/20230321142315_add_column_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddColumnToUsers < ActiveRecord::Migration[6.1]
+  def change
+    rename_column :users, :password, :password_digest
```

Comment:
We do not need to do this right, you can just make changes in the previous migration. because this is not yet merged right?